### PR TITLE
Add details on how to contribute

### PR DESCRIPTION
**Pull request by jerriep**

### CODE REVIEW

**Comment by tparnell8** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ __Adding social authentication to your application is a breeze__ and just requir
 
 See [https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/tree/dev/samples/Mvc.Client](https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/tree/dev/samples/Mvc.Client) for a complete sample __using MVC 6 and supporting multiple social providers__.
 
+## Contributing
```
Comment (PinpointTownes) [inline README.md]: Have you considered merging the `contribute` and `contributions` sections?

**Comment by PinpointTownes** [comment]
Except my minor remark, that's perfect :+1:

File: README.md (hunk 1 above)
Comment (jerriep) [inline README.md]: You mean in the readme.md file? Have a "Contributing" section and then also list actual contributors in that section?

Comment (PinpointTownes) [inline README.md]: Yeah, having two sections for the same things looks weird. Listing the contributors is also a good idea :+1:

Comment (jerriep) [inline README.md]: Sure, I can merge the two together

**Comment by jerriep** [comment]
OK, I merged the two sections and added an alphabetical list of contributors

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    
+If you want to add a new provider, look at what providers are available for you to pick up in [the issues](https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/issues). If you see one you want to work on, leave a comment stating that you are picking it up so we don't get people working on the same things. If you don't see the provider you want to work on listed, then just create an issue for it and then leave a comment that you’re implementing it.
+
+## Using the Yeoman generator
+
+To create a new provider, we have created a Yeoman generator to create the basic scaffolding for you. To install it please ensure you have completed the following:
+
+* Install NodeJS from [their website](https://nodejs.org/en/download/).
+* Install Yeoman by running `npm install -g yo`
+* Next install the generator by running `npm install -g generator-aspnet-oauth`
```
Comment (tparnell8) [inline CONTRIBUTING.md]: this can be combined with the line above `npm install -g yo generator-aspnet-oauth`

Comment (jerriep) [inline CONTRIBUTING.md]: Sure, but maybe people have Yeoman already installed. Also I would like to differentiate between installing Yeoman and whatever generators.

**Comment by PinpointTownes** [comment]
Merged, thanks! https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/commit/385393e3db05730456b91d36b92362d1e36eca53

**Pull request closed without merging**